docs/package.json

```diff
@@ -39,7 +39,7 @@
 		"start": "hugo server"
 	},
 	"dependencies": {
-		"@fluid-tools/api-markdown-documenter": "^0.8.3",
+		"@fluid-tools/api-markdown-documenter": "^0.9.0",
```

Comment:
I recently merged a PR that bumps this to `0.10.1`, FYI. You may have a couple minor merge conflicts. Let me know if you have any questions :)